<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="menuBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>15</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>LOGO</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>391</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="allButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>查看全部</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dashButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>仪表盘</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="distriButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>任务分配</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="imitateButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>22</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>导入模拟数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>791</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QPushButton" name="addPushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>添加服务器</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>451</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>25</number>
      </property>
      <item row="1" column="0">
       <widget class="QGroupBox" name="storBox">
        <property name="title">
         <string/>
        </property>
        <widget class="QProgressBar" name="storProBar">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>90</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>58</number>
         </property>
        </widget>
        <widget class="QLabel" name="storLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>5</y>
           <width>150</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Storage</string>
         </property>
        </widget>
        <widget class="QFrame" name="storFrame">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>113</y>
           <width>212</width>
           <height>40</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>161</width>
            <height>18</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="storLayout">
           <item>
            <widget class="QLabel" name="storgbLabel">
             <property name="text">
              <string>GiB</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="diskLabel">
             <property name="text">
              <string>Disk</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="vituLabel">
             <property name="text">
              <string>GiBSwap</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QLabel" name="diskIcon">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>55</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="machBox">
        <property name="title">
         <string/>
        </property>
        <widget class="QProgressBar" name="memoProBar">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>90</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>69</number>
         </property>
        </widget>
        <widget class="QLabel" name="machLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>5</y>
           <width>150</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Machine</string>
         </property>
        </widget>
        <widget class="QFrame" name="machFrame">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>113</y>
           <width>211</width>
           <height>40</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>161</width>
            <height>18</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="machLayout">
           <item>
            <widget class="QLabel" name="memogbLabel">
             <property name="text">
              <string>Gib</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="styLabel">
             <property name="text">
              <string>DDR3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="procsLabel">
             <property name="text">
              <string>Procs</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QLabel" name="memoIcon">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>55</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="grapBox">
        <property name="title">
         <string/>
        </property>
        <widget class="QGraphicsView" name="grapView">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>431</width>
           <height>131</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="clocBox">
        <property name="title">
         <string/>
        </property>
        <widget class="QFrame" name="clocFrame">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>181</width>
           <height>80</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>134</width>
            <height>42</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="duriLayout">
           <item>
            <widget class="QLabel" name="hLabel">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Hours</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mLabel">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Mins</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sLabel">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Seconds</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="procBox">
        <property name="title">
         <string/>
        </property>
        <widget class="QProgressBar" name="cpuProBar">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>90</width>
           <height>25</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
        <widget class="QLabel" name="procLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>5</y>
           <width>150</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>16</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Processor</string>
         </property>
        </widget>
        <widget class="QFrame" name="procFrame">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>113</y>
           <width>212</width>
           <height>40</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QWidget" name="layoutWidget_6">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>161</width>
            <height>18</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="procLayout">
           <item>
            <widget class="QLabel" name="coreLabel">
             <property name="text">
              <string>Cores</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ghzLabel">
             <property name="text">
              <string>GHz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="bitLabel">
             <property name="text">
              <string>64-bit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QLabel" name="cpuIcon">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>55</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>300</width>
       <height>430</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="allWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>791</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QTableView" name="allTableView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>741</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>menuBox</zorder>
   <zorder>allWidget</zorder>
   <zorder>pageWidget</zorder>
  </widget>
  <action name="actiondsA">
   <property name="text">
    <string>dsA</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
